packages/maskbook/src/plugins/ITO/UI/RemindDialog.tsx

```diff
@@ -0,0 +1,140 @@
+import { Typography, Link, Checkbox, makeStyles, createStyles, FormControlLabel } from '@material-ui/core'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { useStylesExtends } from '../../../components/custom-ui-helper'
+import type { ERC20TokenDetailed, EtherTokenDetailed } from '../../../web3/types'
+import classNames from 'classnames'
+import ActionButton from '../../../extension/options-page/DashboardComponents/ActionButton'
+import { TokenIcon } from '../../../extension/options-page/DashboardComponents/TokenIcon'
+import { formatEthereumAddress } from '../../Wallet/formatter'
+import { resolveLinkOnEtherscan } from '../../../web3/pipes'
+import type { ChainId } from '../../../web3/types'
+import { ClaimStatus } from './ClaimGuide'
+import { useState } from 'react'
+
+const useStyles = makeStyles((theme) =>
+    createStyles({
+        reminderText: {
+            color: '#FF5555',
+            marginTop: theme.spacing(1),
+            marginBottom: theme.spacing(1.5),
+        },
+        reminderTextLast: {
+            marginBottom: 0,
+        },
+        tokenWrapper: {
+            display: 'flex',
+            alignItems: 'center',
+            marginTop: theme.spacing(2),
+            padding: '1rem 2rem',
+            background: theme.palette.mode === 'dark' ? '#17191D' : '#F7F9FA',
+            borderRadius: '15px',
+        },
+        tokenIcon: {
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            width: 39,
+            height: 39,
+        },
+        tokenTextWrapper: {
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+            height: '45px',
```

Comment:
@zhouhanseng  FYI you can use integer directly here.

Reply:
yep